src/js/toast.js

```diff
@@ -0,0 +1,9 @@
+import iziToast from 'izitoast';
+import 'izitoast/dist/css/iziToast.min.css';
+
+export function showToast(header, message) {
```

Comment:
we should handle different kind of messages error, success, info. Also I think all messages should occur in topRight

Also if we have more than 2 params I would prefer use object of parameters. { type, header, message }